src/clients/rust/Cargo.toml

```diff
@@ -2,14 +2,30 @@
 name = "tigerbeetle"
 version = "0.1.0"
 edition = "2021"
+description = "TigerBeetle client for Rust"
+license = "Apache-2.0"
+authors = ["TigerBeetle, Inc"]
+homepage = "https://tigerbeetle.com"
+repository = "https://github.com/tigerbeetle/tigerbeetle"
+documentation = "https://docs.rs/tigerbeetle"
```

Comment:
docs.rs is such a gem of a Rust ecosystem, love how we just have great-looking, familiar docs for free!